source/isaaclab/isaaclab/assets/articulation/articulation.py

```diff
@@ -824,6 +824,42 @@ def write_joint_armature_to_sim(
         # set into simulation
         self.root_physx_view.set_dof_armatures(self._data.joint_armature.cpu(), indices=physx_env_ids.cpu())
 
+    def write_joint_friction_coefficients_to_sim(
+        self,
+        joint_friction_coeff: torch.Tensor | float,
+        joint_dynamic_friction_coeff: torch.Tensor | float,
+        joint_viscous_friction_coeff: torch.Tensor | float,
+        joint_ids: Sequence[int] | slice | None = None,
+        env_ids: Sequence[int] | None = None,
+    ):
+        if int(get_version()[2]) < 5:
```

Comment:
`get_version` is expensive since it reads a file and does a bunch of operations to extract substrings. Would it be possible to use simulation context directly instead?

> https://isaac-sim.github.io/IsaacLab/main/source/api/lab/isaaclab.sim.html#isaaclab.sim.SimulationContext.get_version

Reply:
https://github.com/isaac-sim/IsaacLab/pull/3318/commits/4f325b0f623fccd4e92bc116e906653b0e104dd8 implements what you suggested